### ISSUE TITLE
Tooltip goes behind container

### ISSUE THREAD

**Issue by yusdacra** [iced-rs/iced#749]
https://user-images.githubusercontent.com/19897088/109094767-98747480-772b-11eb-9961-de154360ba0e.mp4

I assume this has something to do with the layer changes done. There should be a way to control where it is rendered, or `Tooltip`s should always be rendered on top of everything.

**Comment by yusdacra** [comment]
https://github.com/yusdacra/iced/commit/c91d59e3c958ae13ba73976405101475fc7883c2 tried messing around with layers here. Added an `on_top` boolean to `Primitive::Clip` to make it so that layer is rendered on top of other layers which don't have `on_top == true`. Even after doing so the tooltip still gets rendered like in the video. I made sure that the tooltip layer is above everything, yet it gets rendered under it.
